docs/ops/banking-engine/B1-LANGGRAPH-RUNBOOK.md

```
@@ -0,0 +1,166 @@
+```
+
+Or if already cloned: `git pull origin agent/factory/bankingengine/b0b1-sandbox`.
+
+### 3.2 Run the 1-node sandbox graph
+
+```bash
+python services/banking-engine/graph_sandbox.py
+```
+
+Expected output (reply text will vary):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)